interceptors/src/unit-test/java/uk/gov/moj/sdt/interceptors/out/FaultOutboundInterceptorTest.java

```diff
@@ -99,18 +82,34 @@ private GenericDao getMockedGenericDao(final ServiceRequest serviceRequest) {
      * #handleMessage(org.apache.cxf.binding.soap.SoapMessage)}.
      */
     @Test
-    public void testHandleMessage() {
+    void testHandleMessage() {
         SdtContext.getContext().setServiceRequestId(1L);
         final SoapMessage soapMessage = getDummySoapMessageWithFault();
         final FaultOutboundInterceptor faultOutboundInterceptor = new FaultOutboundInterceptor();
         final ServiceRequest serviceRequest = new ServiceRequest();
-        faultOutboundInterceptor.setServiceRequestDao(getMockedGenericDao(serviceRequest));
+        when(mockServiceRequestDao.fetch(ServiceRequest.class, 1L)).thenReturn(serviceRequest);

```

Comment:
In the previous version the getMockedGenericDao() method also invoked mockServicerequestDao.persist().  Should this also be expected here?  It is a void method.